From 13f5b09fd8d69eab3dcb2af58b5da4157f176d04 Mon Sep 17 00:00:00 2001
From: Money <563628989@qq.com>
Date: Tue, 1 Nov 2016 16:03:05 +0800
Subject: [PATCH] =?UTF-8?q?=E6=96=B0=E6=89=93=E4=BA=86=E4=B8=80=E4=B8=AA?=
 =?UTF-8?q?=E8=A1=A5=E4=B8=811.5.1?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

---
 studySourceTree/studySourceTree/AppDelegate.m | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/studySourceTree/studySourceTree/AppDelegate.m b/studySourceTree/studySourceTree/AppDelegate.m
index 756f2c2..aef363b 100644
--- a/studySourceTree/studySourceTree/AppDelegate.m
+++ b/studySourceTree/studySourceTree/AppDelegate.m
@@ -51,6 +51,8 @@
     
     // 建立1.5.3的变基测试功能
     // 完成1.5.3的变基测试功能
+    
+    // 新打了一个补丁1.5.1
     return YES;
 }
 
-- 
2.8.1

