diff --git a/studySourceTree/studySourceTree/AppDelegate.m b/studySourceTree/studySourceTree/AppDelegate.m
index 756f2c2..fe8150f 100644
--- a/studySourceTree/studySourceTree/AppDelegate.m
+++ b/studySourceTree/studySourceTree/AppDelegate.m
@@ -55,7 +55,7 @@
 }
 
 - (void)applicationWillResignActive:(UIApplication *)application {
-    
+    // 在1.0.1上打了一个10的补丁
     // 在1.0.1上打了一个1.0.1.3的补丁
     // Sent when the application is about to move from active to inactive state. This can occur for certain types of temporary interruptions (such as an incoming phone call or SMS message) or when the user quits the application and it begins the transition to the background state.
     // Use this method to pause ongoing tasks, disable timers, and throttle down OpenGL ES frame rates. Games should use this method to pause the game.
